diff --git a/types/@netlify/functions/index.d.ts b/types/@netlify/functions/index.d.ts
deleted file mode 100644
index bd6d733..0000000
--- a/types/@netlify/functions/index.d.ts
+++ /dev/null
@@ -1,29 +0,0 @@
-export interface HandlerEvent {
-  rawUrl: string;
-  rawQuery: string | null;
-  path: string;
-  httpMethod: string;
-  headers: Record<string, string | undefined>;
-  queryStringParameters?: Record<string, string | undefined> | null;
-  body?: string | null;
-  isBase64Encoded?: boolean;
-}
-
-export interface HandlerContext {
-  functionName: string;
-  clientContext?: unknown;
-  identity?: unknown;
-}
-
-export interface HandlerResponse {
-  statusCode: number;
-  headers?: Record<string, string | undefined>;
-  multiValueHeaders?: Record<string, string[]>;
-  body?: string;
-  isBase64Encoded?: boolean;
-}
-
-export type Handler = (
-  event: HandlerEvent,
-  context: HandlerContext
-) => Promise<HandlerResponse> | HandlerResponse;
